code/modules/clothing/spacesuits/rig/modules/specific/ai_container.dm

```diff
@@ -129,12 +134,16 @@
 	if(!..())
 		return 0
 
+	if(ai_locked)
+		to_chat(usr, "\The [src] gives an error code response.")
```

Comment:
SPAN_WARNING

Reply:
Several other naked warning messages below as well.

Reply:
My eyes have been opened to a whole new feature I didn't know we have. The tedium of manual SPANs is at an end.